internal/repo/repo_structure_exists.go

```diff
@@ -108,11 +107,21 @@ func checkMissingFiles(listOfFiles []string) []string {
 	return missingFiles
 }
 
-// Function to format output message containing missing files.
-func fmtMessage(startingMessage string, listOfFiles []string) string {
-	message := startingMessage
-	for _, missingFile := range listOfFiles {
-		message += missingFile + " "
+func CreateFiles(files []string) {
```

Comment:
Either do not export these functions, since it is only used in this package, or create a separate filesystem (or better name) package